feat: document ScannableResources and ClearScannableResources

RebeccaStevens opened the pull request:
Accompanying PR for https://github.com/Nogg-aholic/ContentLib/pull/25

RebeccaStevens committed:
feat: document ScannableResources and ClearScannableResources

budak7273 commented on modules/ROOT/pages/Features/Patching.adoc:
```diff
@@ -210,6 +210,25 @@ However, since `Schematic_Alternate_Coal2_C` is of type Alternate and this patch
 
+To unlock scannable resources, you can use the `ScannableResources` field.
+Simply specific the resource you want to unlock and the type of node that should be scanned for.
+Here's an example schematic patch that unlocks the "Uranium" schematic when the "Biocoal" schematic is unlock:
```
Description does not match the example

budak7273 commented on modules/ROOT/pages/Features/Patching.adoc:
````diff
@@ -210,6 +210,25 @@ However, since `Schematic_Alternate_Coal2_C` is of type Alternate and this patch
+}
+```
+
+To unlock geysers, use `Geyser` as the node type and `Desc_Geyser_C` as the resource.
````
```suggestion
To unlock geyser scanning, use `Geyser` as the node type and `Desc_Geyser_C` as the resource.
```

budak7273 committed:
Additional resource scan examples, suggest some generic schematic visual kits instead of items for schematic kits

RebeccaStevens committed:
Update modules/ROOT/pages/Features/Patching.adoc

Co-authored-by: Rob B <computerguy440+gh@gmail.com>

budak7273 committed:
Switch example to Sulfur Node, fix wording.
The wiki claims the scanner does not support looking for just wells? https://satisfactory.wiki.gg/wiki/Resource_Well
